apps/explorer/src/components/ui/PageHeader.tsx

```diff
@@ -35,13 +37,14 @@ export function PageHeader({
     loading,
     after,
     status,
+    showCopyButton = true,
```

Comment:
I think this prop isn’t being used, right? also, could we hide the copy icon if there's an alias? Right now, the header shows two copy buttons, so it feels a bit redundant :see_no_evil: wdyt?
![image](https://github.com/user-attachments/assets/5d06c90c-0650-480e-89b5-652f1843447d)